api/src/rustic_ai/api_server/guilds/service.py

```diff
@@ -24,6 +32,21 @@ def create_guild(self, metastore_url: str, guild_spec: GuildSpec) -> str:
 
         guild = GuildBuilder.from_spec(guild_spec).bootstrap(metastore_url)
 
+        messaging_config = GuildHelper.get_messaging_config(guild_spec)
+        messaging = MessagingInterface(guild.id, messaging_config)
+        msg_id = GemstoneGenerator(random.randint(10, 100)).get_id(priority=Priority.HIGH)
+        system_agent = AgentTag(id="api_server", name="API Server")
+
+        message = Message(
+            id_obj=msg_id,
+            topics=[GuildTopics.SYSTEM_TOPIC],
+            sender=system_agent,
+            format=get_qualified_class_name(GuildReadyMessage),
+            payload={"guild_id": guild.id, "guild_name": guild_spec.name},
+        )
+
+        messaging.publish(sender=system_agent, message=message)
+
```

Comment:
There is no system_agent. The GuildService should never interact or do anything with the messaging interface. Its purpose is to interact with the db and provide isolation to the controller/router. It is solely an abstraction layer for the http apis on the guild.

Reply:
I agree, Service might not be the right place for it.

@Nihal-Srivastava05 This logic might be better in the bootstrap method in builder or in GuildHelper. Look at it. @Shiti what do you think?